From 97d78ae0846890ca33d4d30f12fc944a62844d92 Mon Sep 17 00:00:00 2001
From: Paul Lemire <paul.lemire@kdab.com>
Date: Wed, 5 May 2021 08:13:05 +0200
Subject: [PATCH 02/15] Don't export QKeyFrame which is fully implemented in
 the header

This otherwise leads to inconsistent dll linkage warnigns on windows.

Change-Id: I4f32f2fd2cc45939187535b8232172b280df7316
Reviewed-by: Mike Krus <mike.krus@kdab.com>
(cherry picked from commit 9d17891737befa4645ffd1e1a1153c314366e059)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/animation/frontend/qkeyframe.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/animation/frontend/qkeyframe.h b/src/animation/frontend/qkeyframe.h
index 87e2812b2..a05e446dd 100644
--- a/src/animation/frontend/qkeyframe.h
+++ b/src/animation/frontend/qkeyframe.h
@@ -47,7 +47,7 @@ QT_BEGIN_NAMESPACE
 
 namespace Qt3DAnimation {
 
-class Q_3DANIMATIONSHARED_EXPORT QKeyFrame
+class QKeyFrame
 {
 public:
     enum InterpolationType : quint8 {
-- 
2.37.0

